apps/src/gamelab/GameLab.js

```diff
@@ -73,7 +73,7 @@ var GameLab = function () {
   this.consoleLogger_ = new JsInterpreterLogger(window.console);
 
   /** @type {JsDebuggerUi} */
-  this.debugger_ = new JsDebuggerUi(this.runButtonClick.bind(this));
+  this.debugger_ = null;
```

Comment:
I had to make this change because the `DebugArea` object throws an error once the debugger is disabled: `debugAreaRoot and codeTextboxRoot are required`
Now, gamelab matches applab and only creates the `JsDebuggerUi` when it creates the UI elements. I agree, it was simpler the other way, but for now, I just want to make them work consistently.